sig-robotics/examples/Generating-Action-Sequences-for-Robot-Task-Plans-using-LLM/.gitmodules

```diff
@@ -0,0 +1,10 @@
+[submodule "src/navigation"]
+	path = src/navigation
+	url = https://github.com/fishros/navigation2.git
```

Comment:
(just out of curiosity) Why this is forked from mainline? can we pull with using specific hash or this includes some specific code or requirements?